lib/pavilion/plugins/commands/result.py

```diff
@@ -91,12 +94,21 @@ def run(self, pav_cfg, args):
             title="Test Results"
         )
 
-    def _get_tests(self, pav_cfg, tests_arg):
-        if not tests_arg:
+    def _get_tests(self, pav_cfg, tests_arg, full_arg):
```

Comment:
The logic of this really bugs me. How about this structure instead:
1. Get the tests based on the tests_arg (or absence thereof).
2. Throw an error if the full argument was given and there's more than one test, or print a warning that we've cut it down to just the first one.